contrib/container-images/centos-8-pypy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -x
+# Builds an ARA API server container image using the latest PyPi packages on CentOS 8.
+build=$(buildah from centos:8)
+
+# Get all updates, install pip, database backends and gunicorn application server
+# This lets users swap easily from the sqlite default to mysql or postgresql just by tweaking settings.yaml.
+# Note: We use the packaged versions of psycopg2 and mysql python libraries so
+#       we don't need to install development libraries before installing them from PyPi.
+buildah run "${build}" -- /bin/bash -c "dnf update -y && dnf install -y epel-release && dnf install -y python3-pip python3-gunicorn python3-psycopg2 && dnf clean all"
+
+# Install PyMySQL from pypi as not yet packaged for epel8
+buildah run "${build}" -- /bin/bash -c "pip3 install PyMySQL"
```

Comment:
unfortunately, django is picky and specifically requires "mysqlclient", which is not the same as PyMySQL: https://adamj.eu/tech/2020/02/04/how-to-use-pymysql-with-django/

Reply:
I just inherited from your f32 script  : "python3-PyMySQL" :-)
and btw I see it available as a module from AppStream : 
python3-PyMySQL.noarch                           0.8.0-10.module_el8.1.0+245+c39af44f                        @AppStream

Reply:
I can't blame you because it is indeed confusing but ``python-mysql`` and ``python-pymysql`` are two different things:

- [python-mysql](https://koji.fedoraproject.org/koji/buildinfo?buildID=1449947) comes from [mysqlclient](https://pypi.org/project/mysqlclient/) on PyPi
- [python-pymysql](https://koji.fedoraproject.org/koji/buildinfo?buildID=1448906) comes from [pymysql](https://pypi.org/project/PyMySQL/) on PyPi

The container build scripts install ``python-mysql`` because that's the version that django supports. ``python-pymysql`` is no bueno.